lang/java/reef-applications/reef-vortex/src/test/java/org/apache/reef/vortex/driver/DefaultVortexMasterTest.java

```diff
@@ -52,9 +54,11 @@
   public void testSingleTaskletNoFailure() throws Exception {
     final VortexFunction vortexFunction = testUtil.newIntegerFunction();
     final VortexWorkerManager vortexWorkerManager1 = testUtil.newWorker();
-    final RunningWorkers runningWorkers = new RunningWorkers(new RandomSchedulingPolicy());
+    final RunningWorkers runningWorkers = new RunningWorkers(new RandomSchedulingPolicy(),
+        Tang.Factory.getTang().newInjector().getInstance(AggregateFunctionRepository.class));
```

Comment:
Should we instantiate those repository for all test cases? If so, could we hide the initialization like others (e.g., testUtil.newAggregationFunctionRepositoy()`)? It may not be very helpful because the class name is already long, though :)